src/HttpClient.php

```diff
@@ -289,14 +60,23 @@ public function __construct($disableTls = false, $cafile = null)
      */
     public function download($url)
     {
-        $context = $this->getStreamContext($url);
-        $result = file_get_contents($url, null, $context);
+        if (extension_loaded('zlib')) {
+            array_push($this->headers, "Accept-Encoding: gzip\r\n");
+        }
+
+        if ($this->tls) {
+            humbug_set_headers($this->headers);
+            $result = humbug_get_contents($url);
+        } else {
+            $context = $this->getNonSslStreamContext($url);
+            $result = file_get_contents($url, null, $context);
```

Comment:
I think we can remove the `--disable-tls` option for now. Let's do this once it becomes a problem.